docs/tools/vdb_table/data/aperturedb.json

```diff
@@ -67,93 +67,93 @@
     "comment": ""
   },
   "embeddings_text": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Applications/semantic_search/website_ingest",
+    "comment": "We use text embeddings in ApertureDB vector store for our documentation chatbot."
   },
   "embeddings_image": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Advanced/Image%20Similarity%20Search",
+    "comment": "Supports multimodal embeddings in general including for videos."
```

Comment:
Again, this is about bundling the models into the database - this passage makes me believe that you do not do that:

". The following query will ingest a single image along with its descriptor, and it must be accompanied by a [blob array](https://docs.aperturedata.io/query_language/Reference/shared_command_parameters/blobs) of the form [ <descriptor feature vector>, <encoded image> ]."